src/Lumi/Components2/Banner.purs

```diff
@@ -0,0 +1,231 @@
+                                    <> props.content
+                                }
+                          ]
+                      }
+                ,  Monoid.guard props.dismissable
+                    $ E.element R.div'
+                      { children:
+                          [ Icon.icon
+                              { style: R.css { fontSize: "12px" }
+                              , type_: Icon.Remove
+                              }
+                          ]
+                      , className: ""
+                      , css: theme # toCSS dismissButtonStyle
+                      , onClick: capture_ $ setVisible \_ -> false
+                      }
+                ]
+            }
+  where
+    defaults :: Record BannerProps
+    defaults =
+      { component: Banner
+      , title: Nothing
+      , content: []
+      , dismissable: false
+      , icon: Nothing
+      }
+
+    dismissButtonStyle :: forall props. PropsModifier props
+    dismissButtonStyle =
+      S._interactive
+      <<< style \(LumiTheme { colors }) ->
+            css
+              { position: str "absolute"
+              , top: int 16
+              , right: int 24
+              , "&:hover": nested $ css
+                  { color: color colors.black1
+                  }
+              }
+
+actionBanner :: Array JSX -> PropsModifier BannerProps
+actionBanner actions f =
+  f >>> \props -> props
+    { content =
+        [ box
+          $ S._column
+          $ style_
+              ( css
+                { flex: str "1" }
+              )
+          $ S.onDesktop (S._row)
+          $ style_
+              ( fold
+                  [ css
+                      { alignItems: str "flex-end"
+                      }
+                  , desktopQuery $ css
+                      -- @NOTE positioning of the action button changes dependent on if there is a banner title or not
+                      { alignItems: if (isJust props.title)
+                          then str "flex-end"
+                          else str "center"
+                      }
+                  ]
+              )
+          $ _ { content =
+                  [ box
+                    $ style_
+                        ( css
+                          { flex: str "1" }
+                        )
+                    $ _ { content = props.content }
+                  , box
+                    $ S._row
+                    $ S._align S.Center
+                    $ S._justify S.End
+                    $ style_
+                        ( fold
+                            [ css
+                                { margin: str "16px 0 0"
+                                , "& :not(:first-child)": nested $ css
+                                    { marginLeft: prop S8
+                                    }
+                                }
+                            , desktopQuery $ css
+                                { margin: str "0 0 0 40px"
```

Comment:
Did you mean `4px` here?

Reply:
No, this is intentional. There is an explicit left margin of 40px from the banner text content in the designs.